sudoku/validate_sudoku.py

```diff
@@ -0,0 +1,119 @@
+# validate sudoku
+# input is a board
+# Return True if board is correct
+# Return False if board is incorrect
+
+
+def validate(board):
+
+    board_len = len(board)
+
+    # # check row numbers for duplicates
+    seen = {}
+    dups = []
+    for row in range(board_len):
+        #print( bo[i])
+        for col in board[row]:
+            
+            if col not in seen:
+                seen[col] = 1
+            else:
+                dups.append(col)
+            
+        seen={}
+        if len(dups) > 0:
+            return False
+    
+    # check columns for duplicates
+    seen = {}
+    dups = []
+    for col in range(board_len):
+        for row in board:
+            print(row[col])
+            if row[col] not in seen:
+                seen[row[col]] = 1
+            else:
+                dups.append(row[col])
+        seen = {}
+        if len(dups) > 0:
+            return False
+        print("--------")
```

Comment:
Stop using print. Use logging module and add logging.debug("") statments so that they can be properly managed.

Reply:
fixe the code